### PR TITLE
Added missing tests

### PR DESCRIPTION
**Pull request by vzarytovskii**
Some additional tests were removed during the revert, readding them

### CODE REVIEW

File: tests/service/Symbols.fs [hunk 1]
```diff
@@ -474,3 +474,48 @@ type Foo =
           (:? FSharpMemberOrFunctionOrValue as setMfv) ->
             Assert.AreNotEqual(getMfv.CurriedParameterGroups, setMfv.CurriedParameterGroups)
         | _ -> Assert.Fail "Expected symbols to be FSharpMemberOrFunctionOrValue"
+        
+    [<Test>]
```
Comment (nojaf) [inline tests/service/Symbols.fs:478]: This is not complete @vzarytovskii.
`module GetValSignatureText =` is still missing.
https://github.com/dotnet/fsharp/pull/15625/files#diff-86de212e8dc5b638058239f2b2b485cd6207f61b3acb2556de8bb7fef15065eaL619-L680

Comment (vzarytovskii) [reply tests/service/Symbols.fs:478]: One test is also failing - 
https://dev.azure.com/dnceng-public/public/_build/results?buildId=344741&view=ms.vss-test-web.build-test-results-tab&runId=7207146&resultId=100376&paneView=debug

File: tests/service/Symbols.fs [hunk 2]
```diff
@@ -474,3 +474,107 @@ type Foo =
...
+            |> List.map (fun su -> su.Symbol)
+
+        match symbols with
+        | [ :? FSharpMemberOrFunctionOrValue as cctor
+            :? FSharpMemberOrFunctionOrValue as ctor
+            :? FSharpEntity as entity  ] ->
+            Assert.AreEqual(".cctor", cctor.CompiledName)
+            Assert.AreEqual(".ctor", ctor.CompiledName)
+            Assert.AreEqual("SR", entity.DisplayName)
+        | s -> Assert.Fail ("Expected symbols, got: " + (sprintf "%A" s))
```
Comment (nojaf) [inline tests/service/Symbols.fs:521]:
```suggestion
    [<Test>]
    let ``Multiple relevant symbols for type name`` () =
        let _, checkResults = getParseAndCheckResults """
// This is a generated file; the original input is 'FSInteractiveSettings.txt'
namespace FSInteractiveSettings
type internal SR () =
    static let mutable swallowResourceText = false
    /// If set to true, then all error messages will just return the filled 'holes' delimited by ',,,'s - this is for language-neutral testing (e.g. localization-invariant baselines).
    static member SwallowResourceText with get () = swallowResourceText
                                        and set (b) = swallowResourceText <- b
    // END BOILERPLATE
"""

        let symbols =
            checkResults.GetSymbolUsesAtLocation(5, 16, "type internal SR () =", [ "SR" ])
            |> List.map (fun su -> su.Symbol)

        match symbols with
        | [ :? FSharpEntity as entity
            :? FSharpMemberOrFunctionOrValue as ctor ] ->
            Assert.AreEqual(".ctor", ctor.CompiledName)
            Assert.AreEqual("SR", entity.DisplayName)
        | s -> Assert.Fail ("Expected symbols, got: " + (sprintf "%A" s))
```

File: tests/service/Symbols.fs [hunk 3]
```diff
@@ -474,3 +474,105 @@ type Foo =
...
+    let ``Includes attribute for parameter`` () =
+        assertSignature
+            "val a: [<B>] c: int -> int"
+            """
+module Telplin
+type BAttribute() =
+    inherit System.Attribute()
+let a ([<B>] c: int) : int = 0
+"""
+            (7, 5, "let a ([<B>] c: int) : int = 0", "a")
```
Comment (nojaf) [inline tests/service/Symbols.fs:578]:
```suggestion
            (5, 5, "let a ([<B>] c: int) : int = 0", "a")
```

File: tests/service/Symbols.fs [hunk 4]
```diff
@@ -474,3 +474,105 @@ type Foo =
...
+            "new: builder: ImmutableArray<'T>.Builder -> ImmutableArrayViaBuilder<'T>"
+            """
+module Telplin
+open System
+open System.Collections.Generic
+open System.Collections.Immutable
+type ImmutableArrayViaBuilder<'T>(builder: ImmutableArray<'T>.Builder) =
+    class end
+"""
+            (8, 29, "type ImmutableArrayViaBuilder<'T>(builder: ImmutableArray<'T>.Builder) =", ".ctor")
```
Comment (nojaf) [inline tests/service/Symbols.fs:565]:
```suggestion
            (6, 29, "type ImmutableArrayViaBuilder<'T>(builder: ImmutableArray<'T>.Builder) =", ".ctor")
```

**Comment by vzarytovskii** [comment]
Covered by Eugene's PR

**Pull request closed without merging**